internal/systemd/files_unix.go

```diff
@@ -51,7 +55,7 @@ func Files(unsetEnv bool) []*os.File {
 	}
 
 	nfds, err := strconv.Atoi(os.Getenv("LISTEN_FDS"))
-	if err != nil || nfds == 0 {
+	if err != nil || nfds <= 0 {
```

Comment:
Note: I used the latest version of this file; this change comes from https://github.com/coreos/go-systemd/commit/de1b3a891531d046c05640a265e060c8917a3e37